docs/architecture.md

```diff
@@ -21,52 +21,52 @@ This leads to constant development and DevOps/MLOps work.
 
 Furthermore, as your project scales, you need greater computation power or GPUs, and you need to access large-scale data sets.
 This cannot work on laptops.
-You need a way to seamlessly run your code on a remote cluster and automatically scale it out.
+You need a way to seamlessly run your code on a remote cluster and automatically scale it.
 
 <a id="why-mlrun"></a>
 ## Why MLRun?
 
 When running ML experiments, you should ideally be able to record and version your code, configuration, outputs, and associated inputs (lineage), so you can easily reproduce and explain your results.
 The fact that you probably need to use different types of storage (such as files and AWS S3 buckets) and various databases, further complicates the implementation.
 
-Once you finished development you want to serve models online or build real-time pipelines without having to refactor or 
+Once the development is complete, it's time to serve models online or build real-time pipelines without having to refactor or 
 re-implement the logic, or call an army of developers to help.
 
 Wouldn't it be great if you could write the code once, using your preferred development environment and simple "local" semantics, and then run it as-is on different platforms?
-Imagine a layer that automates the build process, execution, data movement, scaling, versioning, parameterization, outputs tracking, deployment to production, monitoring, and more.
+Imagine having a layer that automates the build process, execution, data movement, scaling, versioning, parameterization, outputs tracking, deployment to production, monitoring, and more.
 A world of easily developed, published, or consumed data or ML "functions" that can be used to form complex and large-scale offline or real-time ML pipelines.
 
-In addition, imagine a marketplace of ML functions that includes both open-source templates and your internally developed functions, to support code reuse across projects and companies and thus further accelerate your work.
+In addition, imagine a marketplace of ML functions that includes both open-source templates and your internally developed functions, to support code for reuse across projects and companies and thus further accelerate your work.
 
-<b>This is the goal of MLRun, Simplify & Accelerate Time To Production.</b>
+<b>This is the goal of MLRun - Simplify & Accelerate Time To Production.</b>
```

Comment:
The hyphen should be replaced with an em dash and the following text shouldn't be capitalized; (the entire sentence is already emphasized using bold text). Please change to the following:
```suggestion
<b>This is the goal of MLRun &mdash; simplify & accelerate time to production.</b>
```